x-pack/plugins/apm/public/components/app/TransactionDetails/Distribution/index.tsx

```diff
@@ -52,11 +53,31 @@ interface Props {
 
 export class Distribution extends Component<Props> {
   public formatYShort = (t: number) => {
-    return `${t} ${unitShort(this.props.urlParams.transactionType)}`;
+    return i18n.translate(
+      'xpack.apm.transactionDetails.transactionsDurationDistributionChart.unitShortLabel',
+      {
+        defaultMessage:
+          '{transCount} {transType, select, request {req.} other {trans.}}',
+        values: {
+          transCount: t,
+          transType: this.props.urlParams.transactionType as string
+        }
+      }
+    );
   };
 
   public formatYLong = (t: number) => {
-    return `${t} ${unitLong(this.props.urlParams.transactionType, t)}`;
+    return i18n.translate(
+      'xpack.apm.transactionDetails.transactionsDurationDistributionChart.unitLongLabel',
+      {
+        defaultMessage:
+          '{transType, select, request {{transCount, plural, =0 {# request} one {# request} other {# requests}}} other {{transCount, plural, =0 {# transaction} one {# transaction} other {# transactions}}}}',
```

Comment:
Wow, that syntax is hard on the eyes...

Reply:
yep, you're right.
I divided into two messages 2ab892d02c9c7ea7490c7d2d5e7fbbe4dc3724fe